try:
    print(a)
except Exception as e:
    print(f'Error: {e}')


# try:
#     print('a')
# except NameError:
#     print('Error: variable a not defined')

# else:
#     print("no error")

# finally:
#     print('Always')